static_src/components/navbar.jsx

```diff
@@ -127,9 +123,6 @@ export class Nav extends React.Component {
                     })}
                   </ul>
                 </li>
-                <li className={ this.styler('marketplace') }>
```

Comment:
Actually nevermind, cg-site will still need it, will do so in the cg-style code.